redirects.next.js

```diff
@@ -181,6 +181,11 @@ module.exports = (async () => {
     '/upgrade-guides/1-0.html': '/language/upgrade-guides/1-0',
     '/upgrade-guides/index.html': '/language/upgrade-guides',
     '/language/resources/provisioners': '/language/resources/provisioners/syntax',
+    '/enterprise/before-installing/cluster-architecture': '/enterprise/admin/infrastructure/active-active',
+    '/enterprise/install/cluster-aws': '/enterprise/admin/infrastructure/active-active',
+    '/enterprise/install/cluster-azure': '/enterprise/admin/infrastructure/active-active',
+    'content/enterprise/install/cluster-custom': '/enterprise/admin/infrastructure/active-active',
```

Comment:
@laurapacilio Looks like this line and the one below have a rogue `content` prefix :)

Reply:
heh thank you - fixed!!